src/java/com/puppetlabs/http/client/impl/ClientMetricFilter.java

```diff
@@ -0,0 +1,11 @@
+package com.puppetlabs.http.client.impl;
+
+import com.codahale.metrics.MetricFilter;
+import com.codahale.metrics.Metric;
+
+public class ClientMetricFilter implements MetricFilter{
+
+    public boolean matches (String name, Metric metric) {
+        return name.startsWith("puppetlabs.http-client");
+    }
```

Comment:
I think this is fine but we should chat briefly about what happens if multiple Puppet Servers are all trying to export this data to graphite.  I *think* the graphite export code will give us a way to namespace / differentiate the metrics between servers, but we should double-check.